.github/workflows/run-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Run tests
+
+on:
+  push:
+  pull_request:
+
+env:
+  # global env to all steps
+  TOX_WORKERS: -n2
+
+jobs:
+  run-test:
+    name: ${{ matrix.python-version }}-${{ matrix.sqlalchemy }}
+    runs-on: "ubuntu-latest"
+    strategy:
+      # run this job using this matrix, excluding some combinations below.
+      matrix:
+        python-version:
+          - "3.9"
+        sqlalchemy:
+          - sqla_release
+          - sqla_master
+
+      fail-fast: false
+
+    # steps to run in each job. Some are github actions, others run shell commands
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: ${{ matrix.architecture }}
```

Comment:
I don't see architecture in the matrix?

Reply:
indeed it's not, but it works in any case.

it's probably a leftover from a file I used as templace